code/public/index.html

```diff
@@ -13,7 +14,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>Technigo React App</title>
+    <title>The Rabbit Hole Labyrinth</title>
   </head>
 
```

Comment:
I think I saw Annika post on Slack about og images.  I found these two resources to be helpful in generating og tags.
https://metatags.io/
https://postimages.org/